### DISCUSSION TITLE
[Deathdusk modpack] Some item icons are invisble in the REI crafting panel

### DISCUSSION THREAD

**Discussion by jony-test** [shedaniel/RoughlyEnoughItems#889] [Q & A]
[pic](https://i.imgur.com/2Ty7luT.jpeg)

Can someone please advise? Is it related to this mod?

**Comment by shedaniel** [comment]
Are you able to see these items in the creative mode menu?

If so, please get back to me because this is a bug.

If it is also invisible in the creative menu, please report the issue to the corresponding mod.

If it is simply missing from the creative menu, then I think we can look at it more in depth.

**Comment by jony-test** [comment]
Not only does it also show up in the creative menu, but it seems that in the case of single player worlds even in survival there seems to be no issue.
[creative single player world](https://i.imgur.com/ZLYBlmO.jpeg)
[survival single player world](https://i.imgur.com/mhwqIje.jpeg)

I've also noticed while investigating issues with the modpack that there's similar broken behavior with the Traveller's Backpack, which also involves Cloth Config (i noticed the modpack has 2 very close versions of the same thing, and the older one of them has been marked by the author as UNUSED)

Maybe the problems are related and it can help shed some light, here's the link to the issue [backpack issue](https://github.com/Tiviacz1337/Travelers-Backpack/issues/397)

**Reply by shedaniel** [reply]
Alrighty, do you have logs to share after trying to view the invisible items in REI?
